src/components/feed/AnswerInputForm.js

```diff
@@ -0,0 +1,90 @@
+import { useEffect, useState } from "react";
+import "../../styles/AnswerInputForm.css";
+
+const AnswerInputForm = ({
+    data,
+    isEdit,
+    onPostAnswer,
+    onEditAnswer,
+    answerContent,
+    setAnswerContent,
+}) => {
+    const initialValue = answerContent || [];
+
+    const [inputValue, setInputValue] = useState(initialValue);
+    // const [isSubmitting, setIsSubmitting] = useState(false);
+    // const [submittingError, setSubmittingError] = useState(null);
+    const [active, setActive] = useState(false);
+
+    const handleChange = (e) => {
+        setInputValue(e.target.value);
+        setActive(e.target.value.trim() !== "");
+    };
+
+    const handlePostAnswer = async (e) => {
+        e.preventDefault();
+        const answerData = {
+            questionId: data.id,
+            content: inputValue,
+            isRejected: false,
+            team: "6-12",
+        };
+        const result = await onPostAnswer(data.id, answerData);
+        setAnswerContent(inputValue);
+    };
+
+    const handleEditAnswer = async (e) => {
+        e.preventDefault();
+        const id = data.answer?.id;
+        const editAnswerData = {
+            content: inputValue,
+            isRejected: false,
+        };
+        const result = await onEditAnswer(id, editAnswerData);
+        setAnswerContent(inputValue);
+    };
```

Comment:
```suggestion
   const handleSubmit = async (e) => {
        e.preventDefault();
        let formData = {
            content: inputValue,
            isRejected: false,
        };
        if (isEdit) {
            const result = await onEditAnswer(data.answer?.id, formData);
        } else {
            formData = {
                ...formData,
                questionId: data.id,
                team: "6-12",
            };
            const result = await onPostAnswer(data.id, formData);
        }
        setAnswerContent(inputValue);
    };
```

이런식으로 하나의 함수로 바꿔서 조건문을 이용해서 처리할수 있겠네요 !